std/format.d

```diff
@@ -3906,6 +3906,29 @@ if (isPointer!T && !is(T == enum) && !hasToString!(T, Char))
     }
 }
 
+/**
+   SIMD vectors are formatted as arrays.
+ */
+void formatValue(Writer, V, Char)(auto ref Writer w, auto ref V val, const ref FormatSpec!Char f)
+if (isSIMDVector!V)
+{
+    formatValue(w, val.array, f);
+}
+
+@safe pure unittest
+{
+    import core.simd;
+    static if (is(int4))
```

Comment:
Use `float4` is it is available on all platforms supporting SIMD instructions. Integer SIMD support is much more rare.